<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="options_listWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>5</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Layout</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QCheckBox" name="saveLayout_checkBox">
                <property name="text">
                 <string>Save dimensions</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Components</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QCheckBox" name="markLowStock_checkBox">
                <property name="text">
                 <string>Mark low stock components</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="showContainers_checkBox">
                <property name="text">
                 <string>Show containers on table</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="containerPage">
          <layout class="QVBoxLayout" name="containerPage_verticalLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetFixedSize</enum>
             </property>
             <item>
              <widget class="QLineEdit" name="container_lineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="addContainer_toolButton">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="toolTip">
                <string>Add Container</string>
               </property>
               <property name="text">
                <string notr="true">a</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/img.qrc">
                 <normaloff>:/img/container_add.png</normaloff>:/img/container_add.png</iconset>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
               <property name="arrowType">
                <enum>Qt::NoArrow</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="removeContainer_toolButton">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="toolTip">
                <string>Remove Container</string>
               </property>
               <property name="text">
                <string notr="true">r</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/img.qrc">
                 <normaloff>:/img/container_delete.png</normaloff>:/img/container_delete.png</iconset>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="packagePage">
          <layout class="QVBoxLayout" name="packagePage_verticalLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetFixedSize</enum>
             </property>
             <item>
              <widget class="QLineEdit" name="package_lineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="addPackage_toolButton">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="toolTip">
                <string>Add Package</string>
               </property>
               <property name="text">
                <string notr="true">a</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/img.qrc">
                 <normaloff>:/img/package_add.png</normaloff>:/img/package_add.png</iconset>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
               <property name="arrowType">
                <enum>Qt::NoArrow</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="removePackage_toolButton">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="toolTip">
                <string>Remove Package</string>
               </property>
               <property name="text">
                <string notr="true">r</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/img.qrc">
                 <normaloff>:/img/package_delete.png</normaloff>:/img/package_delete.png</iconset>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="labelPage">
          <layout class="QVBoxLayout" name="labelPage_verticalLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="primaryLabel_widget" native="true">
             <layout class="QVBoxLayout" name="primaryLabel_verticalLayout">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="sizeConstraint">
                 <enum>QLayout::SetFixedSize</enum>
                </property>
                <item>
                 <widget class="QLineEdit" name="primaryLabel_lineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="addPrimaryLabel_toolButton">
                  <property name="font">
                   <font>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="toolTip">
                   <string>Add Primary Label</string>
                  </property>
                  <property name="text">
                   <string notr="true">a</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../resources/img.qrc">
                    <normaloff>:/img/label_add.png</normaloff>:/img/label_add.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>false</bool>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                  <property name="arrowType">
                   <enum>Qt::NoArrow</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="removePrimaryLabel_toolButton">
                  <property name="font">
                   <font>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="toolTip">
                   <string>Remove Primary Label</string>
                  </property>
                  <property name="text">
                   <string notr="true">r</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../resources/img.qrc">
                    <normaloff>:/img/label_delete.png</normaloff>:/img/label_delete.png</iconset>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="secondaryLabel_widget" native="true">
             <layout class="QVBoxLayout" name="secondaryLabel_verticalLayout">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="sizeConstraint">
                 <enum>QLayout::SetFixedSize</enum>
                </property>
                <item>
                 <widget class="QLineEdit" name="secondaryLabel_lineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="addSecondaryLabel_toolButton">
                  <property name="font">
                   <font>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="toolTip">
                   <string>Add Secondary Label</string>
                  </property>
                  <property name="text">
                   <string notr="true">a</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../resources/img.qrc">
                    <normaloff>:/img/label_add.png</normaloff>:/img/label_add.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>false</bool>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                  <property name="arrowType">
                   <enum>Qt::NoArrow</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="removeSecondaryLabel_toolButton">
                  <property name="font">
                   <font>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="toolTip">
                   <string>Remove Secondary Label</string>
                  </property>
                  <property name="text">
                   <string notr="true">r</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../resources/img.qrc">
                    <normaloff>:/img/label_delete.png</normaloff>:/img/label_delete.png</iconset>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="manufacturerPage">
          <layout class="QVBoxLayout" name="manufacturerPage_verticalLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetFixedSize</enum>
             </property>
             <item>
              <widget class="QLineEdit" name="manufacturer_lineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="addManufacturer_toolButton">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="toolTip">
                <string>Add Manufacturer</string>
               </property>
               <property name="text">
                <string notr="true">a</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/img.qrc">
                 <normaloff>:/img/manufacturer_add.png</normaloff>:/img/manufacturer_add.png</iconset>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
               <property name="arrowType">
                <enum>Qt::NoArrow</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="removeManufacturer_toolButton">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="toolTip">
                <string>Remove Manufacturer</string>
               </property>
               <property name="text">
                <string notr="true">r</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/img.qrc">
                 <normaloff>:/img/manufacturer_delete.png</normaloff>:/img/manufacturer_delete.png</iconset>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Product">
          <widget class="QPushButton" name="PoductCheck_pushButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>140</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Check</string>
           </property>
          </widget>
          <widget class="QPushButton" name="PoductReduce_pushButton">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>140</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Reduce</string>
           </property>
          </widget>
          <widget class="QPushButton" name="PoductAdd_pushButton">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>140</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>80</y>
             <width>111</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Test Coskun </string>
           </property>
          </widget>
          <widget class="QPushButton" name="PoductOpenFile_pushButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>30</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Open File</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="PoductFileAdr_lineEdit">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>30</y>
             <width>151</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
